doc/calling-javascript.md

```diff
@@ -176,6 +176,32 @@ or as
 js.Dynamic.literal("foo" -> 42, "bar" -> "foobar")
 {% endhighlight %}
 
+## Literal object construction using an scala object interface
+Sometimes for a nicer interface, Literal objects can be implimented using 
+a trait interface. The following JavaScript object:
+
+{% highlight javascript %}
+{foo: 42, bar: "foobar"}
+{% endhighlight %}
+
+can be implemented using following code:
+
+{% highlight scala %}
+trait MyObject extends js.Object {
+  val foo: Int = js.native
+  val bar:String= js.native
+}
+//and an scala object for easy access
```

Comment:
Empty line after trait, make this a doc comment and simply comment the object (no inline text in code).